.github/workflows/release.yml

```diff
@@ -30,169 +30,21 @@ jobs:
           persist-credentials: false
           fetch-depth: 0 # Used for conventional commit ranges
 
-      ## This step installs node and sets up several matchers (regex matching for Github
-      ## Annotations). See
-      ## https://github.com/actions/setup-node/blob/25316bbc1f10ac9d8798711f44914b1cf3c4e954/src/main.ts#L58-L65
-      - uses: actions/setup-node@v2.4.0
+      - uses: Workday/canvas-kit-actions/install@v1
         with:
-          node-version: 14.x
-          registry-url: https://registry.npmjs.org
+          node_version: 14.x
 
-      ## The caching steps create a cache key based on the OS and hash of the yarn.lock file. A
-      ## cache hit will copy files from Github cache into the `node_modules` and `.cache/cypress`
-      ## folders. A cache hit will skip the cache steps
-      - name: Cache node modules
-        id: yarn-cache
-        uses: actions/cache@v2
+      - uses: Workday/canvas-kit-actions/release@v1
         with:
-          path: node_modules
-          key: ${{ runner.os }}-node-modules-hash-${{ hashFiles('yarn.lock') }}
+          gh_token: ${{ secrets.GITHUB_TOKEN }}
+          gh_rw_token: ${{ secrets.GH_RW_TOKEN }}
+          publish_token: ${{ secrets.NPM_CI_PUBLISH_TOKEN }}
+          chromatic_project_token: ${{ secrets.CHROMATIC_APP_CODE }}
+          version: ${{ inputs.version || 'patch' }}
```

Comment:
`Workday/canvas-kit-actions/release@v1` is replaced and refactored release action. It can take version from dispatch input (if we need to run release manually with specific tag) or patch as a default value for automatic releases after branch pushes.